src/main/java/com/spotifyapi/dto/TokensDTO.java

```diff
@@ -11,5 +11,6 @@ public class TokensDTO {
 
     private String accessToken;
     private String refreshToken;
+    private String accessCode;
```

Comment:
Do you use accessCode somewhere? Does it change?

Reply:
Yes, i used it for store access code in DB